source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -55,10 +55,10 @@ const HeaderList AWSLambdaFilter::HeadersToSign =
          Http::Headers::get().ContentType});
 
 AWSLambdaFilter::AWSLambdaFilter(Upstream::ClusterManager &cluster_manager,
-                                 Api::Api &api,
+                                 Api::Api &api, bool should_propagate_origin,
```

Comment:
super nit, add new parameter on its own line

Reply:
after looking further, i see some examples in upstream envoy where they do this too. i think this is fine so long as we abide by the column width rules from upstream